tools/RepositoryManagement/DevHome.RepositoryManagement/Strings/en-us/Resources.resw

```diff
@@ -121,4 +121,124 @@
     <value>Repositories</value>
     <comment>Name of the tool to help manage repositories on the user's machine.</comment>
   </data>
+  <data name="ConfigurationFileNameFormat" xml:space="preserve">
+    <value>{0} file</value>
+    <comment>File filter example for chossing where to save a configuration file.</comment>
+  </data>
+  <data name="DeleteRepositoryDialogTitle" xml:space="preserve">
+    <value>Would you like to delete this repository?</value>
+    <comment>Message asking the user to confirm deleting the repository.</comment>
+  </data>
+  <data name="DeleteRepositoryDialogContent" xml:space="preserve">
+    <value>Deleting a repository means it will be permanently removed in File Explorer and from your PC.</value>
+    <comment>Message explaining what will happen if the repository is deleted.</comment>
+  </data>
+  <data name="Yes" xml:space="preserve">
+    <value>Yes</value>
+    <comment>Confirmation word</comment>
+  </data>
+  <data name="Cancel" xml:space="preserve">
+    <value>Cancel</value>
+    <comment>Cancel</comment>
+  </data>
+  <data name="LocateRepositoryDialogTitle" xml:space="preserve">
+    <value>Can not find {0}</value>
+    <comment>{Locked="{0}"} {0} name of the repository.  Title for not able to find a repository.</comment>
+  </data>
+  <data name="LocateRepositoryDialogContent" xml:space="preserve">
+    <value>Can not find {0} at {1}.  Do you know where it is?</value>
```

Comment:
Instead of "Do you know where it is?" can we rely on the button text informing the user that there's an action they can take to fix this? I'm assuming it's the string below. I think "Locate with File Explorer", although the more I think about it I wonder if it implies Dev Home will be doing the locating. Maybe "Select repository root directory"?